com.github.itext.i7j_rups.json

```
@@ -0,0 +1,120 @@
+{
+    "id": "com.github.itext.i7j_rups",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "command": "rups",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk11"],
+    "finish-args": [
+        "--env=PATH=/app/jre/bin:/app/bin:/usr/bin",
+        "--env=JAVA_HOME=/app/jre",
+        "--env=JAVA=/app/jre/bin/java",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
```

```suggestion
        "--filesystem=xdg-documents",
        "--filesystem=xdg-download",
```

I'm not sure if it shouldn't be `home` though.